bundler/spec/dependabot/bundler/update_checker/parent_dependency_resolver_spec.rb

```diff
@@ -0,0 +1,109 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+require "shared_contexts"
+require "dependabot/dependency"
+require "dependabot/dependency_file"
+require "dependabot/bundler/update_checker/parent_dependency_resolver"
+
+RSpec.describe Dependabot::Bundler::UpdateChecker::ParentDependencyResolver do
+  include_context "stub rubygems compact index"
+
+  let(:resolver) do
+    described_class.new(
+      dependency_files: dependency_files,
+      repo_contents_path: repo_contents_path,
+      credentials: [{
+        "type" => "git_source",
+        "host" => "github.com",
+        "username" => "x-access-token",
+        "password" => "token"
+      }]
+    )
+  end
+  let(:dependency_files) { [gemfile, lockfile] }
+  let(:repo_contents_path) { nil }
+
+  let(:dependency) do
+    Dependabot::Dependency.new(
+      name: dependency_name,
+      version: current_version,
+      requirements: [],
+      package_manager: "bundler"
+    )
+  end
+  let(:dependency_name) { "dummy-pkg-a" }
+  let(:current_version) { "1.0.1" }
+  let(:target_version) { "2.0.0" }
+
+  let(:gemfile) do
+    Dependabot::DependencyFile.new(content: gemfile_body, name: "Gemfile")
+  end
+  let(:lockfile) do
+    Dependabot::DependencyFile.new(content: lockfile_body, name: "Gemfile.lock")
+  end
+  let(:gemfile_body) { fixture("ruby", "gemfiles", "subdep_blocked_by_subdep") }
+  let(:lockfile_body) do
+    fixture("ruby", "lockfiles", "subdep_blocked_by_subdep.lock")
+  end
+
+  describe "#blocking_parent_dependencies" do
+    subject(:blocking_parent_dependencies) do
+      resolver.blocking_parent_dependencies(
+        dependency: dependency,
+        target_version: target_version
+      )
+    end
+
+    it "returns the right array of updated dependencies" do
```

Comment:
```suggestion
    it "returns the right array of blocking dependencies" do
```

Nit: I didn't think this was updating anything.